src/dbt/kipptaf/models/assessments/intermediate/int_assessments__all_assessments_aligned.sql

```diff
@@ -0,0 +1,378 @@
+with
+    students as (

```

Comment:
drop students from here and join to students on reporting views downstream

Reply:
The purpose of this int table is to align all the state assessments, including the appropriate demographic info so we can use it in other views. I was planning to use this retrofit the STAT dash, and Walters needs it for the All The New DDI Things, and for Deanslist. Students is being used to customize the demographic info to align with how the state assessment demo proficiency rates need to be reported (FL comes from students, NJ comes from the assessment data). It made sense to me to leave the demographics we need there because our current individual state assessments tables also contain demographic data, and this table is supposed to be the combination of all of these in a way that we can directly use for other things.

Reply:
I see. Thinking we should move this under `tableau/` then. If we're setting up a reporting view to be used multiple dashboards, it would make sense to have an intermediate view there

Reply:
I'd even suggest creating a tableau student enrollments int view that handles all of the common transformations like state_studentnumber etc and we can eventually refactor other views to use it.

Reply:
Oohhh... I love that idea!